src/Core/src/Platform/Android/LocalizedDigitsKeyListener.cs

```diff
@@ -19,13 +20,11 @@ public class LocalizedDigitsKeyListener : NumberKeyListener
 
 		static char GetDecimalSeparator()
 		{
-			if (!(NumberFormat.Instance is DecimalFormat format))
+			if (!(NumberFormat.Instance is DecimalFormat))
```

Comment:
Could you create some related test/s?

Reply:
I couldn't find a way to enter a comma using the numeric keyboard. Please share your suggestion regarding this

Reply:
Could be possible using the `EnterText` method from the Appium helper extension methods?

Reply:
@jsuarezruiz,
If I use the EnterText extension, it passes a string, which is not my case. When the decimal separator is a comma, it should be pressed from the numeric keyboard. Passing it as a string is not related to my fix.